cli/script/command-executor.ts

```diff
@@ -2,14 +2,18 @@
 
 import * as base64 from "base-64";
 import * as chalk from "chalk";
+var childProcess = require("child_process");
 import * as fs from "fs";
```

Comment:
you removed `fs` package from package.json and still are using here. I suppose you didn't clean your binaries that is why it worked fine for you, however this would fail, wouldn't it? Is there someplace else where we have this in our project?